src/main/java/com/avispl/symphony/dal/communicator/management/zoom/rooms/ZoomRoomsCommunicator.java

```diff
@@ -0,0 +1,581 @@
+                    break;
+                case "account_email":
+                    statistics.put("Account Email", value);
+                    break;
+                case "room_name":
+                    statistics.put("Room Name", value);
+                    break;
+                case "platform":
+                    statistics.put("Platform", value);
+                    break;
+            }
+        });
+    }
+
+    /**
+     * Get map of parameters describing the currently connected cameras,
+     * including information like selected/non-selected, name, id, alias etc
+     *
+     * @return {@link Map} containing parameters mapped out of the string feed like:
+     *
+     *      *s Video Camera Line 1 id: 00#8&2cc2822b&0&0000#{65e8773d-8f56-11d0-a3b9-00a0c9223196}\global
+     *      *s Video Camera Line 1 Name: Logi Rally Camera
+     *      ......
+     *      *s Video Camera Line 2 combinedDevice: off
+     *      *s Video Camera Line 2 numberOfCombinedDevices: 0
+     *      *s Video Camera Line 2 ptzComId: -1
+     *
+     * @throws Exception during ssh communication
+     */
+    private Map<String, String> retrieveCameraLineParameters() throws Exception {
+        String cameraLine = execute(ZSTATUS_CAMERA_LINE, "*s Video Camera Line");
+        return parseZoomRoomsProperties(cameraLine, "*s Video Camera Line");
+    }
+
+    /**
+     * Parse properties parameters based on the provided start of the line,
+     * so the expected format of data provided here is:
+     * Video Camera Line 1 Name: Camera Name
+     * .....
+     * Video Camera Line 2 Name: Camera Name
+     *
+     * @param response           string from Zoom Rooms CLI
+     * @param propertyIdentifier supposed start of the string
+     * @return map of a format <Video Camera Line 1 Name: Camera Name, Video Camera Line 2 Name: Camera Name....>
+     */
+    private Map<String, String> parseZoomRoomsProperties(String response, String propertyIdentifier) {
+        Map<String, String> parameters = new HashMap<>();
+        Arrays.asList(response.split("\n")).forEach(s -> {
+            if (s.startsWith(propertyIdentifier)) {
+                String[] values = s.split(":");
+                parameters.put(values[0].replace("*s ", ""), values[1].replace("\r", "").trim());
+            }
+        });
+        return parameters;
+    }
+
+    /**
+     * Join meeting that is currently in progress or start a scheduled one.
+     *
+     * @param value - contains dialString of a format %meetingNumber%.%meetingPassword%@zoomcrc.com
+     *              password is optional.
+     * @return meetingId if operation is successful
+     */
+    private String joinStartMeeting(String value) throws Exception {
+        boolean hasPassword;
+        String meetingNumber;
+        String meetingPassword;
+
+        Pattern dialStringPattern = Pattern.compile("(\\d+)\\.?(\\d+)?(@[a-z]+?\\.[a-z]{2,5})");

```

Comment:
I'm not going to reverse engineer what this pattern will compile too and you did not bother to document it. From the quick look, it appears that it will not allow to connect to E.164 addresses, only to SIP ones, but I might be wrong.

Reply:
But haven't we agreed upon omitting SIP/H323 connection at this point? 
I have talked to Vince and he confirmed that this is not needed for now, so I have intentionally removed the SIP/H323 support leaving Zoom<>Zoom communication. If the pattern gets it match - we suggest it's a zoom SIP address (since we can't rely on a domain at this point also). Then, for zoom meetingNumber/password the call will succeed, but for anything else it would be either skipped, or, if the pattern did match, but false-positively, for non-zoom SIP address - the call will still not succeed and the exception will be thrown

Reply:
Please see my email from Nov 25 with subject "Zoom Rooms feeback".
Please understand that there is no such things as "skipped" in the code. Code is supposed to do what it was asked, or throw an error if it cannot do it. There is no third choice.
From the email (originally came and agreed with Vince): "adapter should not contain any logic related to call types and simply dial to provided address all the time". Please check email for details and talk to me if more info is needed.